week_1/CSS.md

```diff
@@ -0,0 +1,98 @@
+
+```
+body {
+  font-family: Arial, sans-serif;
+  background-color: #fff;
+}
+```
+
+Here, we are giving the font family Arial, a background-color of #fff to the body tag. 
+
+Now, what if you want to select a specific part of your HTML? Like suppose a div. You want only that div to be affected, and the rest of the divs must remain the same.
+
+Now, if you do 
+
+```
+div{
+    background-color: #fff
+}
+```
+This will apply this specific style to *all* the divs. To prevent this, we use *classes* and *ids* that we learnt above.
+Any div can be given a id or class, and we can use selectors in CSS to give styling to that specific class alone.
+
+Now, if your HTML file has a div whose class is "city", this is how you will give style to it:
+
+```
+.city {
+    background-color: #fff;
+}
+```
+
+The dot is called a selector. In CSS, selectors are used to select the elements on an HTML page that you want to style. There are a variety of selectors available, each with its own specific use cases. Here is a basic guide to some common selectors in CSS:
+
+- The element selector selects elements based on the element name. For example, p would select all <p> elements on the page.
+- The #id selector selects elements based on their id attribute. For example, #header would select the element with id="header".
+- The .class selector selects elements based on their class attribute. For example, .warning would select all elements with class="warning".
+
+### Popular properties
+
+- `font-family`: This attribute specifies the font family to be used for the text in an element. It is often used to specify the typeface, such as Times New Roman or Arial.
+- `font-size`: This attribute specifies the size of the font to be used for the text in an element. It is often specified in pixels or em units.
+- `color`: This attribute specifies the color of the text in an element. It can be specified using a named color, such as "red", or using a hexadecimal code, such as "#FF0000".
+- `background-color`: This attribute specifies the background color of an element. It works in the same way as the color attribute, but it applies to the element's background instead of its text.
+- `border`: This attribute is used to specify the width, style, and color of an element's border. It is often used to add visual boundaries around elements.
+
+These are just a few examples of the many attributes available in CSS. There are many other features and techniques in CSS, including the ability to apply styles based on the state of an element, the use of media queries to apply different styles for different screen sizes and devices, and the use of CSS preprocessors to write more concise and powerful stylesheets. For more information on these and other topics, you can refer to the resources.
+
```

Comment:
At least one example with HTML and CSS written together with its preview would be nice